From 16d771f33ea2892aefd0abf08121481026ff03bb Mon Sep 17 00:00:00 2001
From: lei qian <lei.qian@amlogic.com>
Date: Thu, 4 Feb 2021 15:40:02 +0800
Subject: [PATCH] global_keys: delete global keys for AOSP [1/1]

PD#SWPL-42673

Problem:
can't start source menu

Solution:
delete golbal keys for AOSP

Verify:
verify by t5d

Signed-off-by: lei qian <lei.qian@amlogic.com>
Change-Id: Ic877fcd06cb1317ae3ab7bfb5814e07dd2171445
---
 .../base/core/res/res/xml/global_keys.xml     | 37 -------------------
 1 file changed, 37 deletions(-)
 delete mode 100644 overlay/frameworks/base/core/res/res/xml/global_keys.xml

diff --git a/overlay/frameworks/base/core/res/res/xml/global_keys.xml b/overlay/frameworks/base/core/res/res/xml/global_keys.xml
deleted file mode 100644
index 4c0ab10..0000000
--- a/overlay/frameworks/base/core/res/res/xml/global_keys.xml
+++ /dev/null
@@ -1,37 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
-/*
-** Copyright 2014, The Android Open Source Project
-**
-** Licensed under the Apache License, Version 2.0 (the "License");
-** you may not use this file except in compliance with the License.
-** You may obtain a copy of the License at
-**
-**     http://www.apache.org/licenses/LICENSE-2.0
-**
-** Unless required by applicable law or agreed to in writing, software
-** distributed under the License is distributed on an "AS IS" BASIS,
-** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-** See the License for the specific language governing permissions and
-** limitations under the License.
-*/
--->
-
-<!-- Mapping of keycodes to components which will be handled globally.
-     Modify this file to add global keys.
-     The key will NOT go to the foreground application and instead only ever be sent via targeted
-     broadcast to the specified component. The action of the intent will be
-     android.intent.action.GLOBAL_BUTTON and the KeyEvent will be included in the intent as
-     android.intent.extra.KEY_EVENT.
--->
-
-<global_keys version="1">
-    <!-- Example format: id = keycode to handle globally. component = component which will handle
-         this key. -->
-    <key keyCode="KEYCODE_DVR" component="com.google.android.tv/.receiver.GlobalKeyReceiver" />
-    <key keyCode="KEYCODE_GUIDE" component="com.google.android.tv/.receiver.GlobalKeyReceiver" />
-    <key keyCode="KEYCODE_TV" component="com.google.android.tv/.receiver.GlobalKeyReceiver" />
-    <key keyCode="KEYCODE_TV_INPUT" component="com.google.android.tv/.receiver.GlobalKeyReceiver" />
-
-    <!-- OEMs might want to add more custom global key handlings for TV specific key codes. -->
-</global_keys>
-- 
2.28.0

